### PR TITLE
AARA-422: RFA-01A - Differentiate Between "Other Adults Residing In Home" and "Adults Regularly Present"

### PR DESCRIPTION
**Pull request by anuroop15**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/AARA-422

#### Brief feature description
-  add new information to add another adult card to differentiate whether adult is residing in home or regularly present. 
 
#### Does this PR block/effect other stories?
- NA

#### Do any commands needs to be run in order to run the project with this PR?
- NA

#### Code coverage percent diff as a result of this PR?
- NA

### CODE REVIEW

File: app/javascript/components/common/OtherAdultsCardField.js [hunk 1]
```diff
@@ -83,12 +85,26 @@ export class OtherAdultsCardField extends React.Component {
...
         </div>
         <div className='col-md-12 other-adult-date-of-birth'>
+
+        </div>
+        <div className='col-md-12'>
+          <YesNoRadioComponent
+            label={'is this an Adult Residing or Regularly Present in the Home?'}
+            idPrefix={this.isResidingInHome}
+            value={adult.is_residing_in_home}
+            selectClassName={'reusable-select'}
```
Comment (ratneshraval) [inline app/javascript/components/common/OtherAdultsCardField.js]: Do you need `{}` around string values?

File: app/javascript/components/common/OtherAdultsCardField.js [hunk 2]
```diff
@@ -83,12 +85,26 @@ export class OtherAdultsCardField extends React.Component {
...
+
                 </div>
               )
             })
           }
         </div>
         <div className='col-md-12 other-adult-date-of-birth'>
+
+        </div>
+        <div className='col-md-12'>
```
Comment (nasar-cambria) [inline app/javascript/components/common/OtherAdultsCardField.js]: I see the Grid class attribute is already coming from YesNoFields. Is this intentional that you want to create an extra column ?

Comment (anuroop15) [reply app/javascript/components/common/OtherAdultsCardField.js]: Yes. As per design, there was a line separator between Relation to Applicants section and below DateField and is residing in home section. So I kept the div as before.

**Comment by anuroop15** [comment]
jenkins retest this please

**Comment by anuroop15** [comment]
jenkins retest this please

**Pull request merged**